aio/content/guide/upgrade-lazy.md

```diff
@@ -303,17 +301,16 @@ Head over to the [main upgrade guide](guide/upgrade) to learn about:
 </div>
 
 
-## Using Ahead-of-Time compilation with hybrid applications
+## Using ahead-of-time compilation with hybrid apps
 
-You can take advantage of Ahead-of-Time (AoT) compilation on hybrid applications just like on any
-other Angular application. The setup for a hybrid application is mostly the same as described in the
-[AoT Compilation](guide/aot-compiler) guide save for differences in `index.html` and `main-aot.ts`.
+You can take advantage of ahead-of-time (AOT) compilation in hybrid apps just like in any
+other Angular app. The setup for a hybrid app is mostly the same as described in the
+[Ahead-of-Time Compilation](guide/aot-compiler) guide save for differences in `index.html` and `main-aot.ts`.
```

Comment:
The same guide is mentioned as `AOT Compilation` a few lines above.

Reply:
K- I think I got them all.